### ISSUE TITLE
ExpectedEvent.with should support all primitive types

### ISSUE THREAD

**Issue by marcphilipp** [gunnarmorling/jfrunit#20]
In addition to `String` and `Duration`, `ExpectedEvent.with` should support (at least) all primitive value types. Currently, calling `with("name", 1)` yields a `IllegalArgumentException`: https://github.com/gunnarmorling/jfrunit/blob/3848a086f60474733a7b3cc73dd0ecc95a856a55/src/main/java/dev/morling/jfrunit/ExpectedEvent.java#L82

**Comment by gunnarmorling** [comment]
Definitely, all supported JFr event attribute types should be supported. PRs welcome of course :)
